workflow-parser/src/model/workflow-template.ts

```diff
@@ -158,7 +159,7 @@ export type WorkflowDispatchConfig = {
 };
 
 export type WorkflowCallConfig = {
-  inputs?: {[inputName: string]: InputConfig};
+  inputs?: {[inputName: string]: InputConfigScalar};
```

Comment:
We could avoid defining a new type and just do an intersection here to demonstrate what's different
```ts
  inputs?: {[inputName: string]: InputConfig & {default?: string | boolean | number | ScalarToken}};
```

Reply:
So I tried this, ditching the inputConfigScalar and the InputTypeScalar and making a intersection as you suggested. I would *expect* that to work based on what I read about intersections, but it still doesn't like that, to whit: 
```Type 'ScalarToken' is not assignable to type 'InputType'.```